cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,388 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/types"
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+)
+
+const MAXTARGETSHOW = 10
+
+func contextTargetsSize(disruption v1beta1.Disruption) ([]string, error) {
+	spec := disruption.Spec
+	labels := spec.Selector.String()
+	level := spec.Level
+	podNamespaces := disruption.ObjectMeta.Namespace
+	rowNames := 1
+
+	if level == types.DisruptionLevelNode {
+		rowNames = 1
```

Comment:
Afaict this value is never set to anything other than `1`, so I think you can remove this `if` block

Reply:
yes another remnant of when I didnt have the namespace value on hand and the node and pod names were on different rows.